metadata_watcher/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Dockerfile extending the generic Python image to run an metadata watcher app.
+
+FROM gcr.io/google_appengine/python-compat
```

Comment:
Got it. Changed to debian8 as base image. It is much smaller than using gcr.io/google_appengine/python.